RELEASE.md

```diff
@@ -25,3 +25,46 @@
 6. Announce the release on the [mailing list](https://groups.google.com/g/jaeger-tracing), [slack](https://cloud-native.slack.com/archives/CGG7NFUJ3), and [twitter](https://twitter.com/JaegerTracing?lang=en).
 
 Maintenance branches should follow naming convention: `release-major.minor` (e.g.`release-1.8`).
+
+## Release managers
```

Comment:
We should clarify what components are mandatory or optional to release. 

It's fine to document this in the release.md https://github.com/jaegertracing/jaeger/blob/master/RELEASE.md. 
The doc already mentions publishing of docs, but it does not force releasing the UI.

Reply:
This *is* the release.md you linked :-) The case you have in mind is what I thought when I wrote this:

> This person will coordinate the required changes and will address any problems that might happen during the release

Required changes here might be in the other repositories, like IDL/lib/UI. Given that it wasn't clear enough, I just updated the paragraph to spell it out.